recipes/spirv-headers/config.yml

```diff
@@ -0,0 +1,4 @@
+---
+versions:
+  "1.5.1.corrected":
```

Comment:
Same here, see my previous comment. I think it is fine to have a proper version name although the url points  to the "corrected" one